src/Microsoft.IIS.Administration/Microsoft.IIS.Administration.csproj

```diff
@@ -54,6 +54,7 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="1.0.*" />
     <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="1.0.*" />
     <PackageReference Include="Microsoft.AspNetCore.Diagnostics" Version="1.0.*" />
+    <PackageReference Include="Microsoft.Web.LibraryManager.Build" Version="1.0.*" />
```

Comment:
@jimmyca15 Have we always had wildcards in dependency versions? It seems super scary not to have build determinism.

Reply:
No that was a recent change. What is the concern?

Reply:
Non-determinism. You can end up with different versions among devs and the CI server.

Reply:
Current version has all the specific versions, including the Microsoft.Web.LibraryManager.Build one.